pkgs/development/compilers/uhc/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, coreutils, fetchgit, m4, libtool, clang, ghcWithPackages,
+  shuffle,
+  binary, hashable, mtl, network, uhc-util, uulib
+}:
+
+let wrappedGhc = ghcWithPackages ( self: [binary hashable mtl network uhc-util uulib] );
```

Comment:
Why is it necessary to pass `binary` here? That library is a core package, i.e. it comes with `ghc` anyway.